docs/commands/rhoas_service-registry_role_add.md

```diff
@@ -22,7 +28,7 @@ rhoas service-registry role add --role=admin --username=joedough
 
 ```
       --instance-id string       ID of the Service Registry instance to be used (by default, uses the currently selected instance)
-      --role string              Role to apply
+      --role admin               Role to apply: admin, `manager`, or `viewer`
```

Comment:
missing \` for admin

Reply:
I think this is engine problem and we can ignore it.

Other chars like " ' _ are safer in this space